### DISCUSSION TITLE
what's the relations between figma ui kit and primevue? Do I need them?

### DISCUSSION THREAD

**Discussion by xiangnanscu** [primefaces/community#1162] [Figma UI Kit] [answered]
Hello. I'm a full stack developer (lua+vue3),  but totally new to UI stuff like figma. I want to speed up my vue development. I need to drag and pull primevue components in some window and then it generates vue code. Do I need to buy `PrimeOne for Figma` and `Figma` ? What's  the file exported by Figma? If it's not a vue file or its template code, what's the meaning for people like me? Thanks.

**Comment by onursenture** [comment, marked as answer]
PrimeOne for Figma is a Figma project that includes the designs of our component libraries in Figma. Currently, it does not have a code generation feature. It is designed to prevent you from redrawing Prime components during the design phase, where you determine what your product will be like. 

Features such as theme generation and code generation are under development. 

You can follow updates about them at https://primevue.org/roadmap/

**Reply by xiangnanscu** [reply]
That's very cool! Thanks.
